packages/core/src/test/awsService/ec2/model.test.ts

```diff
@@ -132,17 +134,19 @@ describe('Ec2ConnectClient', function () {
         it('calls the sdk with the proper parameters', async function () {
             const sendCommandStub = sinon.stub(SsmClient.prototype, 'sendCommandAndWait')
 
-            sinon.stub(SshKeyPair, 'generateSshKeyPair')
-            sinon.stub(SshKeyPair.prototype, 'getPublicKey').resolves('test-key')
-
             const testSelection = {
                 instanceId: 'test-id',
                 region: 'test-region',
             }
-            const mockKeys = await SshKeyPair.getSshKeyPair('fakeDir', 30000)
-            await client.sendSshKeyToInstance(testSelection, mockKeys, 'test-user')
+            const temporaryDirectory = path.join(globals.context.globalStorageUri.fsPath, 'ModelTests')
```

Comment:
In tests, `globalStorageUri` is stubbed with a temp dir in `globalSetup.test.ts`, so I think this may not be needed

But there is a chance that it is only cleared on the "after" instead of "afterEach", and may cause issues in tests: https://github.com/aws/aws-toolkit-vscode/blob/17e951cc594c580cc28df66053c344520f73fdee/packages/core/src/test/globalSetup.test.ts#L77

So maybe you could add:
```ts
await fs.delete(globals.context.globalStorageUri.fsPath, { recursive: true, force: true })
```
around here:
https://github.com/aws/aws-toolkit-vscode/blob/17e951cc594c580cc28df66053c344520f73fdee/packages/core/src/test/globalSetup.test.ts#L105
which I think we should have been doing in the first place

Reply:
If I move the temp keys into the globalStorage itself, then is there any reason I need this to clear on `afterEach`? I would think deleting the temp dir on `after` should be sufficient.

Reply:
If we do not clear globalStorageUri after each test we'll probably have a collision somewhere later.

Lets say you had a test that created an ssh key, then the following test assumed that since it is a new test the globalStorage will not have a key by default. The initial test state bled over in to the next and left some unexpected artifacts

Reply:
I see. Was there any reason we didn't want to do this before? Is it expensive/slow to delete all its contents for every test?

Reply:
No, we haven't used it much in tests before, so it was just an oversight.